From ccffc7f13d8c2fdb0cb45586a8d65545b9f5e6df Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Thu, 14 Nov 2024 14:32:00 +0100
Subject: [PATCH 1398/1695] dts: stm32: remove provider-cells properties for
 stm32mp25x

Removes theses properties:
- #power-domain-cells
- #clock-cells
- #reset-cells

from scmi node, as theses nodes are not resources provider as same as
their equivalent on the Linux side.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git ea25b04d177b1e1c4d892caf2f24a07a875ca3be dts: stm32: remove provider-cells properties for stm32mp25x]
Fixes: 1bd01d3fb264 ("dts: stm32: describe scmi resources for stm32mp25x")
Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: I1b53b162b26291a8f483add177192e946dc7e066
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/418588
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/dts/stm32mp251.dtsi | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp251.dtsi b/core/arch/arm/dts/stm32mp251.dtsi
index 6d440cb98..a0312b372 100644
--- a/core/arch/arm/dts/stm32mp251.dtsi
+++ b/core/arch/arm/dts/stm32mp251.dtsi
@@ -155,7 +155,6 @@
 				power-domains {
 					#address-cells = <1>;
 					#size-cells = <0>;
-					#power-domain-cells = <1>;
 
 					pd_gpu: pd@0 {
 						reg = <PD_SCMI_GPU>;
@@ -172,7 +171,6 @@
 				clocks {
 					#address-cells = <1>;
 					#size-cells = <0>;
-					#clock-cells = <1>;
 
 					clock@0 {
 						reg = <CK_SCMI_ICN_HS_MCU>;
@@ -1103,7 +1101,6 @@
 				resets {
 					#address-cells = <1>;
 					#size-cells = <0>;
-					#reset-cells = <1>;
 
 					reset@0 {
 						reg = <RST_SCMI_C1_R>;
@@ -1212,7 +1209,6 @@
 				clocks {
 					#address-cells = <1>;
 					#size-cells = <0>;
-					#clock-cells = <1>;
 
 					clock@0 {
 						reg = <CK_SCMI_ICN_HS_MCU>;
@@ -2144,7 +2140,6 @@
 				resets {
 					#address-cells = <1>;
 					#size-cells = <0>;
-					#reset-cells = <1>;
 
 					reset@0 {
 						reg = <RST_SCMI_C1_R>;
-- 
2.47.2

